lib/resources/r4/provenance.yaml

```diff
@@ -0,0 +1,123 @@
+---
+name: Provenance
+field_name_base_url: http://hl7.org/fhir/R4/provenance-definitions.html#Provenance
+fields:
+- name: resourceType
+  required: 'Yes'
+  cardinality: 1..1
+  type: string
+  description: The type of the FHIR resource.
+  action: create
+  example: |
+    {
+      "resourceType": "Provenance"
+    }
+  note: resourceType must be Provenance.
+  url: http://hl7.org/fhir/R4/json.html#resources
+
+- name: target
+  required: 'Yes'
+  cardinality: 1..*
+  type: Reference
+  description: The Reference(s) that were generated or updated by the activity described in this resource.
+  action: create
+  example: |
+    {
+      "target": [
+        {
+          "reference": "DocumentReference/881049"
+        }
+      ]
+    }
+  note: |
+    <ul>
+      <li> Only one target may be provided. </li>
+    </ul>
+  url: http://hl7.org/fhir/R4/provenance-definitions.html#Provenance.target
+
+- name: recorded
+  required: 'Yes'
+  cardinality: 1..1
+  type: instant
+  description: The instant of time at which the activity was recorded.
+  action: create
+  example: |
+    {
+      "recorded": "2019-10-25T14:57:43Z"
+    }
+  url: http://hl7.org/fhir/R4/provenance-definitions.html#Provenance.recorded
+
+- name: agent
+  required: 'Yes'
+  cardinality: 1..*
+  type: BackboneElement
+  description: An actor taking a role in an activity for which it can be assigned some degree of responsibility for the activity taking place.
+  action: create
+  example: |
+    {
+      "agent": [
+        {
+          "type": {
+            "coding": [
+              {
+                "system": "http://hl7.org/fhir/us/core/CodeSystem/us-core-provenance-participant-type",
+                "code": "transmitter"
+              }
+            ]
+          },
+          "role": [
+            {
+              "coding": [
+                {
+                  "system": "http://terminology.hl7.org/CodeSystem/contractsignertypecodes",
+                  "code": "SOURCE"
+                }
+              ]
+            }
+          ],
+          "who": {
+            "reference": "Organization/619848"
+          }
+        }
+      ]
+    }
+  note: |
+    <ul>
+      <li> Only one agent may be provided. </li>
+      <li> All `coding` must contain the `system` and `code` fields. </li>
+    </ul>
+  url: http://hl7.org/fhir/R4/provenance-definitions.html#Provenance.agent
+
+- name: entity
+  required: 'No'
+  cardinality: 0..*
+  type: BackboneElement
+  description: An entity used in this activity.
+  action: create
+  example: |
+    {
+      "entity": [
+        {
+          "role": "source",
+          "what": {
+            "reference": "DocumentReference/S-201"
+          }
+        }
+      ]
+    }
+  url: http://hl7.org/fhir/R4/provenance-definitions.html#Provenance.entity
+  children:
+
+  - name: role
```

Comment:
This only shows up under the terminology bindings, is it meant to show up in body fields as well? It looks like some fields like 'required' and 'cardinality' don't get displayed anywhere.

Reply:
Yeah it is only supposed to be in the terminology bindings because of the required binding on role. And yeah they dont get displayed

Reply:
Needed to add fields for backbone elements: https://github.com/cerner/fhir.cerner.com/pull/371/commits/15b404d166e91e3d9a557e0ad922bfbea002f6c6